pdk_regression.yml

```diff
@@ -3,48 +3,7 @@ channels:
   - litex-hub
   - defaults
 dependencies:
-  - _libgcc_mutex=0.1=main
-  - _openmp_mutex=5.1=1_gnu
-  - bzip2=1.0.8=h7b6447c_0
-  - ca-certificates=2022.10.11=h06a4308_0
-  - certifi=2022.12.7=py37h06a4308_0
-  - dbus=1.13.18=hb2f20db_0
-  - expat=2.4.9=h6a678d5_0
-  - fontconfig=2.14.1=h52c9d5c_1
-  - freetype=2.12.1=h4a9f257_0
-  - gdbm=1.18=hd4cb3f1_4
-  - glib=2.69.1=he621ea3_2
-  - gmp=6.2.1=h295c915_3
-  - gst-plugins-base=1.14.0=h8213a91_2
-  - gstreamer=1.14.0=h28cd5cc_2
-  - icu=58.2=he6710b0_3
-  - jpeg=9e=h7f8727e_0
-  - klayout=0.28.3_3_g76e005267=20230116_180937_py37
-  - ld_impl_linux-64=2.38=h1181459_1
-  - libffi=3.4.2=h6a678d5_6
-  - libgcc-ng=11.2.0=h1234567_1
-  - libgomp=11.2.0=h1234567_1
-  - libpng=1.6.37=hbc83047_0
-  - libstdcxx-ng=11.2.0=h1234567_1
-  - libuuid=1.41.5=h5eee18b_0
-  - libxcb=1.15=h7f8727e_0
-  - libxml2=2.9.14=h74e7548_0
-  - ncurses=6.3=h5eee18b_3
-  - openssl=1.1.1s=h7f8727e_0
-  - pcre=8.45=h295c915_0
-  - pip=22.3.1=py37h06a4308_0
-  - python=3.7.16=h7a1cb2a_0
-  - qt=5.9.7=h5867ecd_1
-  - readline=8.2=h5eee18b_0
-  - ruby=3.1.2=h006b21b_1
-  - setuptools=65.6.3=py37h06a4308_0
-  - sqlite=3.40.1=h5082296_0
-  - tk=8.6.12=h1ccaba5_0
-  - tzdata=2022g=h04d1e81_0
-  - wheel=0.37.1=pyhd3eb1b0_0
-  - xz=5.2.10=h5eee18b_0
-  - yaml=0.2.5=h7b6447c_0
-  - zlib=1.2.13=h5eee18b_0
+  - klayout
   - pip
```

Comment:
maybe also add `- python` before `pip` to be a little bit more explicit and be sure we don't default on the python from the parent environment (being host or conda)

Reply:
Done https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/pull/1/commits/1728f6344781d4575c648aa932bab53a77ce07e4

@proppy